### PR TITLE
Feature #241 - Tests for deactivate business

### LINKED ISSUE #46: Deactivate business
As Owner, I want to delete my business, so I can deactivate it.

**Acceptance Criteria**
_Scenario: The Owner wants to delete his business from website_

1. Owner opens business profile and presses "delete business" button.
2. Business disappears from owners profile, if deleted successfully

Backend
{

- DELETE methods is added to API
- Delete business from database

}

Frontend
{

- "Delete business" button
- Pop-up message "Do you really want to delete it?" with buttons Yes and No

}

Tasks

- [x] #240
- [x] #241

### LINKED ISSUE #241: SP: 1 Create tests for deactivating Business

### PR DESCRIPTION
**Pull request by datsiuk-ira**

### CODE REVIEW

File: beauty/api/tests/test_DELETE_method_to_make_business_inactive.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
+"""This module is for testing DELETE method that is used to inactivate Business."""
+
+from django.test import TestCase
+from rest_framework.test import APIClient
+from rest_framework.reverse import reverse
+from api.models import Business
+from .factories import BusinessFactory, CustomUserFactory
+from ..serializers.business_serializers import BusinessGetAllInfoSerializers
```
Comment (pazuzu-ua) [inline beauty/api/tests/test_DELETE_method_to_make_business_inactive.py]: Maybe change to api.serializers.business_serializers ?

File: beauty/api/tests/test_DELETE_method_to_make_business_inactive.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+class TestDeleteBusiness(TestCase):
+    """This class represents a Test case for the making Business inactive.
+
+    All the needed information is submitted in the setUp method.
+    """
+
+    def setUp(self) -> None:
+        """This method adds needed info for tests."""
+        self.owner = CustomUserFactory(first_name="OwnerUser")
+        self.user1 = CustomUserFactory.create(is_active=True)
```
Comment (pazuzu-ua) [inline beauty/api/tests/test_DELETE_method_to_make_business_inactive.py]: why user1? Just user will be fine =)
Also, no need to assign is_active=True, it is automatically True. You are not doing it in 19th row anyway

File: beauty/api/tests/test_DELETE_method_to_make_business_inactive.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    def test_delete_business(self):
+        """Business can make himself inactive."""
+        response = self.client.delete(
+            path=reverse(
+                "api:business-detail",
+                kwargs={"pk": self.business1.id},
+            ),
+        )
+        self.assertEqual(response.status_code, 200)
+        self.assertFalse(Business.objects.get(pk=1).is_active)
```
Comment (pazuzu-ua) [inline beauty/api/tests/test_DELETE_method_to_make_business_inactive.py]: Why query? You already have self.business1. Don't hard code values

Comment (datsiuk-ira) [reply beauty/api/tests/test_DELETE_method_to_make_business_inactive.py]: It's an accident, I'm sorry

**Review by OleksiiDatsiuk — APPROVED** [review]

**Review by pazuzu-ua — APPROVED** [review]

**Comment by datsiuk-ira** [comment]
Close #241

**Pull request merged**